www/templates/diary/trip_list_item.html

```diff
@@ -1,105 +1,98 @@
+                                <!-- Origin for Trip -->
+                                <div class="diary-street two-lines" ng-click="showDetail($event, trip)">
+                                    <i class="icon ion-location" style="font-size: 16px; left: 0; color: #80D0FF;"></i>
+                                    {{trip.start_display_name}}
+                                    </div>
 
-                    <!-- Row, containing Detail Screen button and Trip Origin/Destination -->
-                    <div class="row" style="padding: 4px">
-                        <div class="col-100" style="width: 100%; height: 70px">
-                            <!-- Origin for Trip -->
-                            <div class="diary-street two-lines" ng-click="showDetail($event, trip)">
-                                <i class="icon ion-location" style="font-size: 16px; left: 0; color: #80D0FF;"></i>
-                                {{trip.start_display_name}}
+                                <!-- Destination for Trip -->
+                                <div class="diary-street two-lines" ng-click="showDetail($event, trip)">
+                                    <i class="icon ion-flag" style="font-size: 16px; left: 0; color: #0088ce;"></i>
+                                    {{trip.end_display_name}}
                                 </div>
-
-                            <!-- Destination for Trip -->
-                            <div class="diary-street two-lines" ng-click="showDetail($event, trip)">
-                                <i class="icon ion-flag" style="font-size: 16px; left: 0; color: #0088ce;"></i>
-                                {{trip.end_display_name}}
                             </div>
                         </div>
                     </div>
                 </div>
-              </div>
 
-              <div class="diary-infos" style="padding-top: 10px; padding-bottom: 10px; width: 100%;">
-                <!-- Mode and Purpose Selection -->
-                <div class="row" style="margin-top: 0px; padding: 4px">
-                    <!-- Assumes that the trip labels have already been
-                    filled in outside the directive to filter properly
-                    e.g. https://github.com/e-mission/e-mission-docs/issues/674#issuecomment-933083710
-                    -->
-                    <div class="col-90">
+                <div class="diary-infos" style="padding-top: 10px; padding-bottom: 10px; width: 100%;">
+                    <!-- Mode and Purpose Selection -->
+                    <div class="row" style="margin-top: 0px; padding: 4px">
+                        <!-- Assumes that the trip labels have already been
+                        filled in outside the directive to filter properly
+                        e.g. https://github.com/e-mission/e-mission-docs/issues/674#issuecomment-933083710
+                        -->
+                        <div class="col-90">
```

Comment:
Extraneous whitespace?

Reply:
The formatting of the HTML was quite poor before this change. Do you want to go back to how it was?

Reply:
I don't want to spend too much time on this, but in what way was the formatting worse? This seems like it basically just moved everything in by 4 spaces/1 tab.

Reply:
some start and close tags did not align, some children elements were not indented, and some `>` were hanging on their own line

before
https://github.com/sebastianbarry/e-mission-phone/blob/9dafcc21f303e60a94250d1c9c608939b2637486/www/templates/diary/trip_list_item.html

after
https://github.com/sebastianbarry/e-mission-phone/blob/6f2100e4e67d092b1d35da4b4d5631f787d4032c/www/templates/diary/trip_list_item.html